performance/pom.xml

```diff
@@ -58,45 +70,42 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>chronos-jmeter-maven-plugin</artifactId>
-                <configuration>
-                    <input>${basedir}/src/jmx</input>
-                    <jMeterRemoteLocation>http://www.apache.org/dist/jmeter/binaries/apache-jmeter-2.8.zip
-                    </jMeterRemoteLocation>
-                    <jmeterhome>${project.basedir}/jmeter</jmeterhome>
-                    <jmeterOutput>${project.basedir}/target/chronos/performancetest</jmeterOutput>
-                    <historydir>${history.directory}</historydir>
-                    <heap>1024m</heap>
-                    <loggc>false</loggc>
-                    <jmeterVariables>
-                        <property>
-                            <name>validator</name>
-                            <value>${validator.version}</value>
-                        </property>
-                    </jmeterVariables>
-                    <options>
-                        <option>${profilingOptions}</option>
-                    </options>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>2.2</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
```

Comment:
Ok, I see. I thought there'd be a dedicated JMH plug-in, but they use the shade plug-in in their archetype, too.